lib/pipe/pipe-helpers.c

```diff
@@ -0,0 +1,63 @@
+#include "pipe-helpers.h"
+
+static BOOL frida_pipe_is_windows_vista_or_greater(void);
+static BOOL frida_pipe_is_windows_8_or_greater(void);
+static BOOL frida_pipe_is_windows_version_or_greater(DWORD major, DWORD minor, DWORD service_pack);
```

Comment:
Looks like the editor may have removed the space before the parentheses on these three prototypes.